doc/cookbook/source/examples/getting_started/passing_features_from_matrix.rst

```diff
@@ -0,0 +1,32 @@
+============================
+Passing features from matrix
+============================
+
+In this getting started guide, we will look at how to pass data random data to a matrix & pass it through a features object .
+
+-------
+Example
+-------
+
+We start off by creating a real matrix and inserting random values. 
+
+.. sgexample:: passing_features_from_matrix.sg:create_matrix
+
+Now we can pass the data from the vector to the features object.
+
+.. sgexample:: passing_features_from_matrix.sg:create_features
+
+----
+NOTE
+----
+
+It is important to note that conventionally, data is stored in a row-major fashion, meaning two row vector of size 3.
```

Comment:
Suggestion: "Note that data is stored in [column major format](https://en.wikipedia.org/wiki/Row-_and_column-major_order), i.e. each column of the matrix corresponds to an observation / feature vector, where each vector consists of a number of variables that is equal to the number of rows of the matrix."
I.e. add the wikipedia link for explanation and for the feature vector

Reply:
And remove the other sentences